### DISCUSSION TITLE
Unable to start previewer

### DISCUSSION THREAD

**Discussion by Varmod** [AvaloniaUI/Avalonia#12282] [General]
Hi,

I'm having trouble getting the previewer to run because it crashes when my app tries to load some resources.
It seems that the previewer process looks for my resources in the project root folder instead of the  /bin/debug/net6.0/ folder. Is this expected behavior? Is there a way to change the path somewhere?

Thanks

**Comment by timunie** [comment]
you can do `Design.IsDesignMode` in your code or ViewModel in order to skip loading these files or define other path.

**Reply by Varmod** [reply]
This doesn't work for me. There are too many file that would have to be skipped. But you're answer works in general. Thanks
